webapp.py

```diff
@@ -0,0 +1,9 @@
+from flask import Flask
+app = Flask(__name__)
+
+@app.route("/hello")
```

Comment:
If I go to localhost:5000 I get 404 not found, only localhost:5000/hello works
this is ok, but after the flask app is served, the stdout says: "Running on http://10.0.2.15:5000/"
And this address does not actually exist, so maybe remove the "/hello" and make it only "/"
Or have both :)